src/libstore/globals.hh

```diff
@@ -716,7 +716,7 @@ public:
 
           - `apple-virt`
 
-            Included on darwin if virtualization is available.
+            Included on Darwin if virtualization is available.
```

Comment:
Maybe better to say "macOS".

Reply:
It think the availability of hardware virtualization is between us and the kernel, so Darwin seems ok.
I agree that macOS could be better if we were somewhat consistent about it. Hard to get rid of `darwin` though, because of `system` strings, especially since flakes.
cc @fricklerhandwerk